jobs/search-alert/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.8
+
+# https://github.com/mozilla-services/Dockerflow/blob/master/docs/building-container.md
+ARG USER_ID="10001"
+ARG GROUP_ID="app"
+ARG HOME="/app"
+
+ENV HOME=${HOME}
+RUN groupadd --gid ${USER_ID} ${GROUP_ID} && \
+    useradd --create-home --uid ${USER_ID} --gid ${GROUP_ID} --home-dir ${HOME} ${GROUP_ID}
+
+WORKDIR ${HOME}
+
+RUN pip install --upgrade pip
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
I'd maybe add `--no-cache-dir` just to reduce the image size a little bit

Reply:
Good idea, we can add it to the template in this repo too.

Reply:
awesome, thanks!